src/ui/components/core/MessageHistory.tsx

```
@@ -37,11 +39,55 @@ export default function MessageHistory({ messages }: MessageHistoryProps) {
+                    <Box key={index} marginY={1} paddingLeft={2}>
+                      <Text color="cyan">{element.content}</Text>
+                    </Box>
+                  );
+                case 'heading':
+                  return (
+                    <Text key={index} bold color={element.level && element.level <= 2 ? "yellow" : "white"}>
+                      {element.content}
+                    </Text>
+                  );
+                case 'mixed-line':
+                  const inlineElements = parseInlineElements(element.content);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)